From 8ea58d163c7367800f5e28d89ed97c598c6b74aa Mon Sep 17 00:00:00 2001
From: Nadav Haklai <nadavh@marvell.com>
Date: Sun, 21 Jul 2013 14:57:05 +0300
Subject: [PATCH 0882/1825] usb: udc: Improve style in mv_udc_main.c

https://github.com/MISL-EBU-System-SW/misl-windriver.git linux-3.4.69-14t2-read
commit d4ddfb56b9f5ce1cbf83ddc91052e37c24af84b6

	- Remove bogus USB_UNDERRUN_WA related code.
	- Move function declarations to appropriate headers.

Change-Id: I23e7a1c1bd710fddf5446207021b3046bda321a7
Signed-off-by: Zbigniew Bodek <zbb@semihalf.com>
Signed-off-by: Nadav Haklai <nadavh@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/2737
Tested-by: Star_Automation <star@marvell.com>
Signed-off-by: Zhong Hongbo <hongbo.zhong@windriver.com>
---
 arch/arm/plat-armada/mv_hal/usb/mvUsb.h |    9 +++++----
 1 files changed, 5 insertions(+), 4 deletions(-)

diff --git a/arch/arm/plat-armada/mv_hal/usb/mvUsb.h b/arch/arm/plat-armada/mv_hal/usb/mvUsb.h
index 514084d..6ffb2a2 100644
--- a/arch/arm/plat-armada/mv_hal/usb/mvUsb.h
+++ b/arch/arm/plat-armada/mv_hal/usb/mvUsb.h
@@ -79,10 +79,11 @@ typedef struct {
 } MV_USB_HAL_DATA;
 
 /* Functions */
-MV_STATUS   mvUsbPllInit(void);
-MV_STATUS   mvUsbHalInit(int dev, MV_BOOL isHost, MV_USB_HAL_DATA *halData);
-MV_U32      mvUsbGetCapRegAddr(int devNo);
-void        mvUsbDevResetComplete(int devNo);
+MV_STATUS	mvUsbPllInit(void);
+MV_STATUS	mvUsbHalInit(int dev, MV_BOOL isHost, MV_USB_HAL_DATA *halData);
+MV_U32		mvUsbGetCapRegAddr(int devNo);
+MV_U32		mvUsbGetCapRegAddr(int devNo);
+void		mvUsbDevResetComplete(int devNo);
 
 #ifdef MV_USB_VOLTAGE_FIX
 MV_U8       mvUsbGppInit(int dev);
-- 
1.7.5.4

